data_measurements/dataset_utils.py

```diff
@@ -48,17 +57,14 @@
 TOT_WORDS = "total words"
 TOT_OPEN_WORDS = "total open words"
 
-_DATASET_LIST = [
-    "c4",
-    "squad",
-    "squad_v2",
-    "hate_speech18",
-    "hate_speech_offensive",
-    "glue",
-    "super_glue",
-    "wikitext",
-    "imdb",
-]
+_CACHE_DATASET_LIST = [dset_info.id for dset_info in list_datasets(author="datameasurements", use_auth_token=HF_TOKEN)]
```

Comment:
Since this is a constant used several times but rewritten as the same string, we may want to define it above (AUTHOR = "datameasurements"), then just use the AUTHOR variable here.